internal/kubernetes/operator/dbusers/dbusers_test.go

```diff
@@ -138,57 +140,53 @@ func Test_buildUserSecret(t *testing.T) {
 	})
 }
 
-type MockUserStore struct {
-	Users map[string][]mongodbatlas.DatabaseUser
-}
-
-func (m *MockUserStore) DatabaseUsers(groupID string, _ *mongodbatlas.ListOptions) ([]mongodbatlas.DatabaseUser, error) {
-	return m.Users[groupID], nil
-}
-
 func TestBuildDBUsers(t *testing.T) {
+	ctl := gomock.NewController(t)
+	defer ctl.Finish()
```

Comment:
newer versions of gomock no longer require the defer, it happen automatically

Reply:
Done